examples/sdk/auth-delegated/web/src/pages/Register.tsx

```diff
@@ -1,20 +1,30 @@
 import { WebAuthn } from '@dfns/sdk-webauthn'
+import { BrowserKeySigner } from '@dfns/sdk-browsersigner'
```

Comment:
I think there was discussions around gathering any features meant for browsers in the same `@dfns/sdk-browser` package or smth ? Where we would have this `BrowserKeySigner` + `WebauthNSigner`, since both are meant to be used in browsers

Reply:
yeah would like to consolidate all the browser specific functionalities into one package and avoid more small packages. 👍

Reply:
Yes makes sense. I'll do it.